pkg/network/driver/common.go

```diff
@@ -103,6 +105,19 @@ func (h *NetworkUtilsHandler) LinkSetHardwareAddr(link netlink.Link, hwaddr net.
 func (h *NetworkUtilsHandler) LinkByName(name string) (netlink.Link, error) {
 	return netlink.LinkByName(name)
 }
+
+func (h *NetworkUtilsHandler) LinkByNames(names []string) (netlink.Link, error) {
```

Comment:
The logic of this function is not trivial, to say the least.
I also do not think such logic should exist in a "handler", it is suppose to be a simple adapter with no business logic.

I think you can just add a wrapper function in the caller code to solve this, with no need to add it here.